server/gokbg3/src/integration-test/groovy/gokbg3/rest/UsersTestSpec.groovy

```diff
@@ -98,35 +115,37 @@ class UsersTestSpec extends AbstractAuthSpec {
   void "test PUT /rest/users/{id}"() {
     def urlPath = getUrlPath()
     // use the bearerToken to write to /rest/user
+    String bodyText = "{data:{id: $altUser.id," +
```

Comment:
Preferably just define a map here and convert it via `body(bodyText as JSON)` later (like records in IntegrationControllerSpec for example).